src/connector/src/parser/mod.rs

```diff
@@ -1013,7 +1014,10 @@ impl SpecificParserConfig {
         let protocol_config = match format {
             SourceFormat::Native => ProtocolProperties::Native,
             SourceFormat::None => ProtocolProperties::None,
-            SourceFormat::Debezium => ProtocolProperties::Debezium,
+            SourceFormat::Debezium => {
+                let debezium_props = DebeziumProps::from(&info.format_encode_options);
+                ProtocolProperties::Debezium(debezium_props)
+            }
```

Comment:
@xiangjinwu I am not sure where to put this code, `ignore_key` seems the first `format` accepting options.